core/src/main/java/org/kohsuke/stapler/StaplerRequest.java

```diff
@@ -25,13 +25,13 @@
 
 import org.apache.commons.beanutils.BeanUtils;
 import org.apache.commons.beanutils.ConvertUtils;
-import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.fileupload2.FileItem;
 
-import javax.servlet.RequestDispatcher;
-import javax.servlet.ServletContext;
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
```

Comment:
Note for myself and other reviewers more than author: this is binary-incompatible (https://github.com/jenkinsci/jenkins/pull/6085#issuecomment-1205147176). As in https://github.com/jenkinsci/jenkins/pull/4848, japicmp may be helpful for summarizing the incompatible changes in the current revision, to make it easier to see whether they are in widely used types.